mungegithub/mungers/nag-flake-issues.go

```diff
@@ -30,7 +30,7 @@ import (
 
 const (
 	flakeNagNotifName = "FLAKE-PING"
-	defaultTimePeriod = 14 * 24 * time.Hour
+	defaultTimePeriod = 4 * 24 * time.Hour
```

Comment:
Add a comment: "Treat unprioritized as P1, to get a human to at least prioritize."